### ISSUE TITLE
Each action add a new line, even if they are dependency a same module

### ISSUE THREAD

**Issue by iahu** [unlight/sublime-import-helper#68]
each action add a new line, even if they are dependency a same module
```js
import React from 'react';
import { useEffect } from 'react'
import { useContext } from 'react'
```

It should be one line
```js
import React,  { useEffect, useContext } from 'react'
```

**Comment by unlight** [comment]
Yes, it's known.
Such syntax is not supported in typescript.
At least shoud be:
```
import React from 'react';
import { useEffect, useContext } from 'react'
```
I'll check what I can do.

**Comment by rafaumeu** [comment]
the same happens when the file is closed, it adds as imports of a depedence in a new line. When should add in the same entry already created previously

**Comment by iahu** [comment]
@rafaumeu  in face, there are another project named [import-js](https://github.com/Galooshi/import-js), just work well in most time.

**Comment by unlight** [comment]
@iahu Should be fixed in version `3.0.0`

**Comment by iahu** [comment]
That’ll be a good news, thanks.
